### ISSUE TITLE
[lighthouse] contrast issue blocking 100% accessibility

### ISSUE THREAD

**Issue by yangwao** [kodadot/nft-gallery#6848]
> For desktop there is a low contrast issue which is preventing 100% accessibility. Might need design intervention if we want to tackle this in future.
image

![image](https://github.com/kodadot/nft-gallery/assets/5887929/6e634971-9094-482a-95e5-1375e29f67d1)

> I noticed on a few components the usage of "aria-role" attribute instead of just "role". I think this may have been a mistake and should be updated to "role"...

we might completely change landing page :)

_Originally posted by @yangwao in https://github.com/kodadot/nft-gallery/issues/6843#issuecomment-1685937150_

**Comment by exezbcz** [comment]
need design for high contrast web?
